input/postgres/relation_stats_ext.go

```diff
@@ -10,10 +10,13 @@ import (
 const citusRelationSizeSQL = `
 SELECT logicalrelid::oid,
        pg_catalog.citus_table_size(logicalrelid)
-  FROM pg_catalog.pg_dist_partition
+	FROM pg_catalog.pg_dist_partition dp
+			 INNER JOIN pg_catalog.pg_class c ON (dp.logicalrelid::oid == c.oid)
+			 INNER JOIN pg_catalog.pg_namespace n ON (c.relnamespace = n.oid)
+ WHERE ($1 = '' OR (n.nspname || '.' || c.relname) !~* $1)
 `
 
-func handleRelationStatsExt(db *sql.DB, relStats state.PostgresRelationStatsMap, postgresVersion state.PostgresVersion) (state.PostgresRelationStatsMap, error) {
+func handleRelationStatsExt(db *sql.DB, relStats state.PostgresRelationStatsMap, postgresVersion state.PostgresVersion, ignoreRegexp string) (state.PostgresRelationStatsMap, error) {
```

Comment:
I think this `ignoreRegexp` needs to be passed to the stmt.Query() call at line 27